vnext/Microsoft.ReactNative/Fabric/ComponentView.cpp

```diff
@@ -399,6 +399,14 @@ bool ComponentView::getAcccessiblityIsReadOnly() noexcept {
   return false;
 }
 
+ToggleState ComponentView::getToggleState() noexcept {
+  return ToggleState::ToggleState_Off;
```

Comment:
Is there a way to set this value from JS?

Reply:
There isn't a direct prop that connects to a toggle value except for the Switch control which has the `value` prop. I believe once we implement accessibilityActions that can used as a way to notify the JS of a toggle event so custom controls can handle toggle behavior, but we don't have a prop right now that stores a toggle value for any component type. Maybe accessibilityValue could be expanded to account for this?

Reply:
Looks like accessibilityState should map to this: https://reactnative.dev/docs/next/accessibility#accessibilitystate

accessibilityActions  "Toggle" does seem like the right thing to do to notify the JS.

Reply:
Yep I agree. I'm going to add that implementation in a separate PR when I add support for accessibilityActions